packages/core/src/providers/apple.ts

```diff
@@ -106,7 +106,8 @@ export interface AppleProfile extends Record<string, any> {
  *
  * #### Configuration
  *
- * Import the provider and configure it in your **Auth.js** initialization file:
+ * Import the provider and configure it in your **Auth.js** initialization file.
+ * Also add the pkceCodeVerifier cookie to the cookies option.
```

Comment:
```suggestion
 * Import the provider and configure it in your **Auth.js** initialization file:
```

This seems unnecessary, since the option is already added via `checks: ["pkce"]`

Reply:
The reason I added the pkce cookie manually is that I get this error when removing it, even if the pkce check is set:
```
sveltekit-auth-app:dev: [auth][error][InvalidCheck]: Read more at https://errors.authjs.dev#invalidcheck
sveltekit-auth-app:dev: [auth][cause]: InvalidCheck: PKCE code_verifier cookie was missing.
```

Could this be an issue with SvelteKitAuth not correctly calling the init function?

Reply:
Sounds like that, yes.